README.md

```diff
@@ -93,8 +91,6 @@ Example: To build for production with parcel, use the folowing commands:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
 👤 **Author1**
```

Comment:
[OPTIONAL] Very well done on the readme. It looks very well built but I think that you can make this section look even more professional by replacing ```Author1``` here with your name.

Reply:
Thanks, @amon-cofie 
I'll surely do that.